lrlex/src/lib/ctbuilder.rs

```diff
@@ -672,7 +672,7 @@ where
                     }
                 }
 
-                has_unallowed_missing = self.warnings_are_errors;
+                has_unallowed_missing |= self.warnings_are_errors;
```

Comment:
In the case where `has_unallowed_missing = true` and `self.warnings_are_errors = false` this would,
unset the `has_unallowed_missing` value back to false.

This was caught by lrpar/cttests/src/ctfails/missing.test